index.html

```diff
@@ -347,7 +384,7 @@ <h2>Contact me</h2>
           </p>
         </div>
 
-        <div class="contact-forms">
+        <div class="contact-forms  footer-items">
           <form class="message-form" action="https://formspree.io/f/1" method="post">
             <ul>
               <li class="input-li">
```

Comment:
- [ ] You are almost there with your styling. I just noticed that you added you should remove the **border** on the `Get In Touch` button to stick to the Figma design. You can only add this property(`border: none;`) in the button styling to fix the issue.

- [ **OPTIONAL** ] **RECOMENDED:** I think it would be better if you use the text `Write your messages here` as a `placeholder="..."` as for now it requires the user to every time deleted the text itself before he writes his message. I know you can do that :muscle: :muscle:.